### PR TITLE
Remove part of #34 since the original issue has been fixed in guardian

### PR DESCRIPTION
**Pull request by fpalluel**
See #39

### CODE REVIEW

File: lib/guardian_db.ex [hunk 1]
```diff
@@ -77,10 +77,10 @@ defmodule GuardianDb do
   @doc """
   After the JWT is generated, stores the various fields of it in the DB for tracking
   """
-  def after_encode_and_sign(_resource, _type, claims, jwt) do
+  def after_encode_and_sign(resource, type, claims, jwt) do
     case Token.create!(claims, jwt) do
-      {:error, _} -> {:error, :token_storage_failure}
-      _           -> :ok
+      { :error, _ } -> { :error, :token_storage_failure }
```
Comment (doomspork) [inline lib/guardian_db.ex]: Could you please remove the spaces inside these tuples?

https://github.com/christopheradams/elixir_style_guide#spaces

**Review by doomspork — CHANGES_REQUESTED** [review]
Thank you @Moxide!  One small change and then we should be good.

File: lib/guardian_db.ex (hunk 1 above)
Comment (fpalluel) [reply lib/guardian_db.ex]: Ok ;-) Also cleaned code in other places in the file.

**Comment by doomspork** [comment]
Thank you @Moxide 😀

**Comment by fpalluel** [comment]
You're welcome ;-)

**Comment by imranismail** [comment]
This needs a dependency bump for guardian. Need to cut out a release for both guardian and guardian_db.

**Comment by doomspork** [comment]
I am going to work on that today or tomorrow @imranismail 👍

**Pull request merged**